content/pages/whoweare.html

```diff
@@ -105,6 +105,22 @@ <h3>Adina Wagner</h3>
       </div>
     </div>
 
+    <div class="row">
+      <h3>Vanessa Sochat</h3>
+      <figure><img src="/theme/img/team/vanessa.png" class="pull-left"/></figure>
+      <div class="col-md-4 col-sm-4 position">Stanford University Research Computing Center, Stanford CA</div>
+      <div class="col-md-5 col-sm-5 contact">
+        <a href="https://github.com/vsoch" class="github">Github</a>
+        <a href="https://twitter.com/vsoch" class="twitter">Twitter</a>
+        <a href="https://www.linkedin.com/in/vsochat/" class="linkedin">LinkedIn</a>
+        <a href="https://vsoch.github.io" class="blog">Blog</a>
+      </div>
+      <div class="col-md-9 col-sm-9 description">
+        Research Software Engineer in the Stanford Research Computing Center. Likes programming,
+        containers, programming, avocados, and open-source development. 
```

Comment:
I guess liking programming twice could be a good thing ;-) would be lovely if you mentioned that you are the one of singularity hub and working with me (it CON) on ... projects

Reply:
okay updated with more details! I didn't add the CON bit because I think this is implied with being under the contributors section on the CON website.

Reply:
sure, but on what projects? ;-) there is a good number of them, so I thought it is valuable to inform visitors about common endeavors.  But that is ok - if you like it as is, I like it too! merging!

Reply:
Honestly Im not a fan of putting micro resumes into a small blob bio, people really just want to know who you are and they can follow links to get details. But you pushed me, so I put more than I typically would.